itl/command-plugins-windows.conf

```diff
@@ -317,3 +317,31 @@ object CheckCommand "users-windows" {
 		}
 	}
 }
+
+object CheckCommand "file-age-windows" {
+        command = [ PluginDir + "/check_file_age.cmd" ]
+
+        arguments = {
+                "file" = {
+                        skip_key = true
+                        order = 0
+                        value = "$file_age_file$"
+                        description = "File name and location"
+                        required = true
+                }
+                "warning" = {
+                        skip_key = true
+                        order = 1
+                        value = "$file_age_warning$"
+                        description = "Warning threshold of file age in seconds"
+                        required = true
+                }
+                "critical" = {
+                        skip_key = true
+                        order = 2
+                        value = "$file_age_critical$"
```

Comment:
Same as above, please rename to `$file_age_win_critical$`

Reply:
Okay I will do that!